### DISCUSSION TITLE
sqlmodel support

### DISCUSSION THREAD

**Discussion by voice1** [awtkns/fastapi-crudrouter#108] [Ideas]
Is there any intent to support SQLModel? [https://github.com/tiangolo/sqlmodel](https://github.com/tiangolo/sqlmodel)

**Comment by awtkns** [comment]
Thanks for bringing this up @voice1 as I was not aware of `sqlmodel` as it is impossible for me to keep track of all ORMs. Looking at it, I think there would be strong intertest to support it going forward given how well it plays with pydantic and fastapi 🚀 

I will create an issue for this to supported in a future release. Is this something you would be potentially interested in PRing?

**Reply by awtkns** [reply]
Issue can be tracked here: https://github.com/awtkns/fastapi-crudrouter/issues/109
